.planning/phases/24-improved-cost-summary/24-03-SUMMARY.md

```
@@ -0,0 +1,105 @@
+  - "External provider costs shown as model links, not prices"
+
+issues-created: []
+
+# Metrics
+duration: 15min
+completed: 2026-01-17
+---
+
+# Phase 24 Plan 3: Update CostDialog UI Summary
+
+**Simplified CostDialog showing Gemini pricing with model links for fal.ai/Replicate instead of unreliable external pricing**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)